packages/flutter_tools/templates/app/android-java.tmpl/app/build.gradle.tmpl

```diff
@@ -51,6 +51,9 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
+            minifyEnabled true
+            useProguard true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
What would be the difference between adding `proguard-android.txt` instead of `proguard-android-optimize.txt`?

Reply:
It looks like `proguard-android-optimize.txt` turns on optimizations that could cause issues on some versions of Dalvik.

https://android.googlesource.com/platform/sdk/+/master/files/proguard-android-optimize.txt#4